packages/zosjobs/src/GetJobs.ts

```diff
@@ -82,6 +82,22 @@ export class GetJobs {
         return GetJobs.getJobsCommon(session, { owner, prefix });
     }
 
+    /**
+     * Get a list of jobs that match various parameters
+     * @static
+     * @param {AbstractSession} session - z/OSMF connection info
+     * @param {string}
+     * @returns {Promise<IJob[]>} - promise that resolves to an array of IJob objects (matching jobs)
+     * @memberof GetJobs
+     */
+    public static async getJobsByParameters(session: AbstractSession, params: IGetJobsParms) {
```

Comment:
Currently we have getJobsByOwnerAndPrefix, this can be the new api that gets jibs by any parameter.